app/helpers/cookies_helper.rb

```diff
@@ -0,0 +1,11 @@
+module CookiesHelper
+  def policy_cookie
+    cookie = cookies['cookies_policy']
+
+    cookie.present? ? JSON.parse(cookie) : {}
+  end
+
+  def usage_enabled?
+    policy_cookie['usage'] == 'true'
+  end
+end
```

Comment:
Do we need this?

Reply:
It's a handy helper that's used in the two partial to decide whether to add the tags to the page or not.
We could just use the test `policy_cookie['usage'] == 'true'` in the partials, but it's just handy to have.